spec/identifiers/isbn_spec.rb

```diff
@@ -126,15 +126,23 @@
 
   context 'when passing prefixes' do
     it 'extracts only prefixed ISBNs' do
-      text = "ISBN:9789992158104  \n ISBN-10 9789971502102 \n IsbN-13: 9789604250592 \n 9788090273412"
+      text = "ISBN:9789992158104  ISBN-10 9789971502102 \n IsbN-13: 9789604250592 \n 9788090273412"
       prefixes = ['IsBn', 'ISBN-10', 'ISBN-13']
 
       expect(described_class.extract(text, prefixes))
         .to contain_exactly('9789992158104', '9789971502102', '9789604250592')
     end
 
+    it 'extracts prefixed ISBNs with special characters, hyphens and unicode' do
```

Comment:
I'd split this test. One test for support of prefixes that contain special characters for regular expressions. And another for ISBNs with hyphens. It is better for tests to only test one feature, not multiple features at the same time.

Reply:
Add another test for ISBNs with spaces

Reply:
What do  you mean by `unicode` in the test description?

Reply:
Unicode dashes. I will split the test and I will explain it better, I thought they were "visible" in the code or something as I saw them different in my editor. My fault @epaune-alt